ios/RNLinksdk.m

```diff
@@ -250,6 +208,89 @@ - (void)dismissLinkViewController {
     self.linkViewDelegate = nil;
 }
 
+- (PLKConfiguration*)getLinkTokenConfiguration:(NSDictionary*)configuration {
+    NSString *linkTokenInput = [RCTConvert NSString:configuration[kRNLinkKitConfigLinkTokenKey]];
+    NSString *oauthRedirectUri = [RCTConvert NSString:configuration[kRNLinkKitConfigOAuthRedirectUriKey]];
+    NSString *oauthNonce = [RCTConvert NSString:configuration[kRNLinkKitConfigOAuthNonceKey]];
+
+    PLKConfiguration* linkConfiguration = [[PLKConfiguration alloc] initWithLinkToken:linkTokenInput];
+
+    if ([oauthRedirectUri length] > 0) {
+        linkConfiguration.oauthRedirectUri = [NSURL URLWithString:oauthRedirectUri];
+    }
+    if ([oauthNonce length] > 0) {
+        linkConfiguration.oauthNonce = oauthNonce;
+    }
+    
+    return linkConfiguration;
+}
+
+- (PLKConfiguration*)getLegacyLinkConfiguration:(NSDictionary*)configuration {
+  // Configuration
```

Comment:
💅 The comment does not seem to carry much information, maybe remove it?

Reply:
⚡